editor/src/templates/editor.tsx

```diff
@@ -431,6 +431,7 @@ export class Editor {
   ): {
     entireUpdateFinished: Promise<any>
   } => {
+    resetRunDomSamplerLimit()
```

Comment:
I wonder if we should have one function here that resets _all_ execution count limited wrapped functions, otherwise there will be a long list here

Reply:
In `LimitExecutionCountOptions` there's a `addToResetArray` property which can be used to put the reset calls into a common array for that kind of usage. Since there's only one and this is partly to demonstrate things, I didn't use that in this PR (other than in the tests). But also I foresee that it's likely we'll have different reset points through the code and not just the one big one here at the start of the dispatch.

Reply:
I have a feeling that we benefit if the first version is more specific to the dispatch flow (and thus we use the resetArray), rather than an overly generic function, but we'll see once we add the second use, I guess

Reply:
Changes from elsewhere discussion made in [0ac7c0f](https://github.com/concrete-utopia/utopia/pull/6353/commits/0ac7c0f616f9068ee4663efde369ddcfabeec8f2).